aspnetcore/toc.yml

```
@@ -968,8 +968,29 @@
           displayName: deploy, publish
           uid: host-and-deploy/iis/protocols
     - name: Kestrel
-      uid: fundamentals/servers/kestrel
       displayName: deploy, publish, server
+      items:
+        - name: Overview
+          displayName: deploy, publish, server
+          uid: fundamentals/servers/kestrel
+        - name: Options
+          displayName: deploy, publish, server
+          uid: fundamentals/servers/kestrel/options
+        - name: Endpoints
+          displayName: deploy, publish, server
+          uid: fundamentals/servers/kestrel/endpoints
```

Prioritize endpoints, everybody needs those.
```suggestion
        - name: Endpoints
          displayName: deploy, publish, server
          uid: fundamentals/servers/kestrel/endpoints
        - name: Options
          displayName: deploy, publish, server
          uid: fundamentals/servers/kestrel/options
```